src/controllers/eventTest.ts

```diff
@@ -1,9 +1,9 @@
 import { Context } from 'koa';
-import EventTesterService from '../services/eventTest/eventTester';
+import { EventTesterService } from '../services/eventTest/eventTester';
```

Comment:
Is there a way to make sure no import has been missed without {} as `default` is being removed ?

Reply:
The `type-script` compiler was throwing error in those cases in the build process. Thats how I verified